scripts/start_port.sh

```diff
@@ -67,8 +68,7 @@ echo "Applying automated fixes"
     sed -i 's/Mathbin\./Mathlib\./g' "$mathlib4_path"
     sed -i '/^import/{s/[.]Gcd/.GCD/g; s/[.]Modeq/.ModEq/g; s/[.]Nary/.NAry/g; s/[.]Peq/.PEq/g; s/[.]Pfun/.PFun/g; s/[.]Pnat/.PNat/g; s/[.]Smul/.SMul/g; s/[.]Zmod/.ZMod/g}' "$mathlib4_path"
 
-    # awk script taken from https://github.com/leanprover-community/mathlib4/pull/1523
-    awk '{do {{if (match($0, "^  by$") && length(p) < 98 && (!(match(p, "^[ \t]*--.*$")))) {p=p " by";} else {if (NR!=1) {print p}; p=$0}}} while (getline == 1) if (getline==0) print p}' "$mathlib4_path" > "$mathlib4_path.tmp"
+    python3 $root_path/scripts/fix-line-breaks.py "$mathlib4_path" "$mathlib4_path.tmp"
```

Comment:
Could also use 
```suggestion
    python3 "$(dirname "${BASH_SOURCE[0]}")"/fix-line-breaks.py "$mathlib4_path" "$mathlib4_path.tmp"
```
(untested)

Reply:
That doesn't work on my system as-is – at that point in the shell script the changed part returns the temporary directory created for the just-ported file.